app/assets/javascripts/discourse/tests/acceptance/topic-edit-timer-test.js

```diff
@@ -133,6 +133,41 @@ acceptance("Topic - Edit timer", function (needs) {
     assert.ok(regex.test(text));
   });
 
+  test("schedule - last custom date and time", async function (assert) {
+    updateCurrentUser({ moderator: true });
+
+    await visit("/t/internationalization-localization");
+    await click(".toggle-admin-menu");
+    await click(".admin-topic-timer-update button");
+
```

Comment:
sorry very much nitpicking here, but I don't know why sometimes you add linebreak between browser actions and sometimes not :p

Reply:
I like to keep you on your toes @jjaffeux :P

Reply:
In seriousness after reviewing these, it is because it is grouped in my head to similar states; the group you highlighted for example is 1. visiting the page and opening the topic timer modal. The next block of actions is 2. click on custom date and time and press save.

Reply:
End of nitpick after this, don't want to waste your time.

Basically I think it's more readable to just do:

- setup
linebreak
- steps
linebreak
- assertion

Anything else will be subject to interpretation IMO. This being said I don't mind with what you did.